Sources/StreamChat/Models/ChatMessage.swift

```diff
@@ -202,6 +202,8 @@ public struct ChatMessage {
     public var readByCount: Int { _readByCount }
 
     @CoreDataLazy internal var _readByCount: Int
+    
+    public let poll: Poll?
```

Comment:
Since it is public and most of the public properties have documentation then I guess it would be nice to have something here as well although it is pretty self-explanatory.